be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -209,10 +209,11 @@ void SinkBuffer::cancel_one_sinker(RuntimeState* const state) {
 }
 
 void SinkBuffer::_update_network_time(const TUniqueId& instance_id, const int64_t send_timestamp,
-                                      const int64_t receive_timestamp) {
-    _last_receive_time = MonotonicNanos();
+                                      const int64_t receive_process_time) {
+    const int64_t receive_response_time = MonotonicNanos();
```

Comment:
renamed to `responsed_timestamp`?

Reply:
or `get_response_timestamp`? the receive is confusing with exch reveiver.

Reply:
Ok, I get it.